### PR TITLE
Lm01/master

### PR DESCRIPTION
**Pull request by gaand**

### CODE REVIEW

File: lib/diagnostic.js [hunk 1]
```diff
@@ -1,62 +1,64 @@
...
-};
+}
 
 // Question 5
-// Write a function that returns the first element
-// for which invoking the callback returns a truthy value
+// Write a function that returns `true`
+// if an only if invoking the callback
+// for every element returns `true`
+// for one and only one element
```
Comment (payne-chris-r) [inline lib/diagnostic.js]:
```diff
-// if an only if invoking the callback
+// if and only if invoking the callback
```

**Review by payne-chris-r — COMMENTED** [review]
Looks good. Single typo. Happy to merge and open an issue, or you can add the `d` and re-push.

File: lib/diagnostic.js (hunk 1 above)
Comment (gaand) [inline lib/diagnostic.js]: Please merge and open an issue.  Thanks.

**Review by payne-chris-r — COMMENTED** [review]
Opened as #159

**Pull request merged**